lxqt-admin-user/CMakeLists.txt

```diff
@@ -4,7 +4,12 @@ project(lxqt-admin-user)
 include_directories (
     ${CMAKE_CURRENT_BINARY_DIR}
 )
-
+if(${CMAKE_SYSTEM_NAME} MATCHES  "FreeBSD")
+    set(OS_FREEBSD 1)
+else()
+    set(OS_FREEBSD 0)
+endif()
+configure_file("system_detect.h.in" "${PROJECT_SOURCE_DIR}/system_detect.h")
```

Comment:
We should use the Qt5 signal/slot connection notation... in that case, everything should work.